doc/release-notes/0.16/0.16.md

```diff
@@ -0,0 +1,155 @@
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/6271">#6271</a></td>
+<td valign="top">Class data sharing for bootstrap classes is enabled by default</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">Class data sharing is enabled by default for bootstrap classes, unless your application is running in a container. You can use the <tt>-Xshareclasses</tt> option to change the default behavior.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/6541">#6541</a></td>
+<td valign="top">New option to share anonymous classes in the shared classes cache</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">Anonymous classes are now stored in the shared classes cache by default so that they are available for ahead-of-time (AOT) compilation. To
+prevent these classes being shared, use the <tt>-XX:-ShareAnonymousClasses</tt> option.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/6234">#6234</a></td>
+<td valign="top">Performance improvements for JVMTI watched fields on IBM Power&reg; systems</td>
+<td valign="top">OpenJDK8 and later (AIX&reg; and Linux on POWER)</td>
+<td valign="top">In earlier releases, performance improvements were introduced for JVMTI watched fields on x86, Linux on Z, and z/OS systems. These
+improvements are now available for AIX and Linux on POWER systems.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/6713">#6713</a></td>
+<td valign="top">Support for Transparent HugePage allocation by default</td>
+<td valign="top">OpenJDK8 and later (Linux on x86)</td>
+<td valign="top">In release 0.15, support was introduced for Transparent HugePage on Linux when the <tt>madvise</tt> (<tt>/sys/kernel/mm/transparent_hugepage/enabled</tt>) setting is set by specifying the <tt>-XX:+TransparentHugePage</tt> option on the command line. This support is now enabled by default for Linux on x86 systems. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/5164">#5164</a></td>
+<td valign="top">New <tt>jcmd</tt> tool</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">For compatibility, OpenJ9 includes an independent implementation of the <tt>jcmd</tt> tool, which can run diagnostic commands on a specified VM or VMs.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/3743">#3743</a></td>
+<td valign="top">Automatically setting an initial heap size by default</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">In release 0.15, a new feature was introduced that allowed OpenJ9 to learn and set an appropriate initial heap size for an application as an alternative to a user manually sizing and setting an <tt>-Xms</tt> value. This feature could be enabled by setting the <tt>-XX:+useGCStartupHints</tt> option on the command line. This
```

Comment:
`-XX:+useGCStartupHints`  Does the option start with a lower case `u` or is this a typo?  We typically ensure they start with a capital

Reply:
The option is `-XX:+UseGCStartupHints`

Reply:
Fixed.